src/main/java/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -42,11 +45,12 @@
   /**
    * Returns a map of unit -> transport. Tries to load units evenly across all transports.
    */
-  public static Map<Unit, Unit> mapTransportsToLoad(final Collection<Unit> units,
-      final Collection<Unit> transports) {
+  public static Map<Unit, Unit> mapTransportsToLoad(final Collection<Unit> units, final Collection<Unit> transports) {
 
-    final List<Unit> canBeTransported = sortByTransportCostDescending(units);
-    final List<Unit> canTransport = sortByTransportCapacityDescendingThenMovesDescending(transports);
+    List<Unit> canBeTransported = CollectionUtils.getMatches(units, Matches.unitCanBeTransported());
+    canBeTransported = sortByTransportCostDescending(canBeTransported);
```

Comment:
Would be nice if we could get rid of the two-step initialization in all these local variables.  Is the reason for doing it in two steps to improve readability, or am I missing some subtle defect in the following statement:

```java
final List<Unit> canBeTransported =
    sortByTransportCostDescending(CollectionUtils.getMatches(units, Matches.unitCanBeTransported()));
```

Reply:
I felt that having the match filter in the sort method limited re-use and hid it so thought it was better this way.

Reply:
Ok, readability is always a good reason. :smile: